README.md

```diff
@@ -215,13 +202,34 @@ Run the web app container using:
 
 # Troubleshooting
 
-<!-- TODO: Add Common Troubleshooting Issues -->
+There is a large amount of user uploaded images in a long running web app
+
+> When running the web app at `http://localhost:8088` an admin page is available at
+> [`http://localhost:8088/cleanup`](http://localhost:8088/cleanup) that allows the user to delete all user uploaded
+> files from the server.
+>
+> [Note: This deletes **all** user uploaded images]
 
-<!-- Idea: Explain cleanup page here? -->
+![Admin UI Screenshot](doc/source/images/cleanup.png)
 
 # Links
 
-<!-- TODO: Add Links -->
+* [Model Asset eXchange (MAX)](https://developer.ibm.com/code/exchanges/models/)
+* [Center for Open-Source Data & AI Technologies (CODAIT)](https://developer.ibm.com/code/open/centers/codait/)
+* [MAX Announcement Blog](https://developer.ibm.com/code/2018/03/20/igniting-a-community-around-deep-learning-models-with-model-asset-exchange-max/)
+
+## Libraries
```

Comment:
Haven't seen this section before in code patterns. I would suggest moving the important ones back up into "Featured Technologies", and removing the rest. I think Bootstrap and Pexels should make the cut.